auraed/src/runtime/cell_service/executables/executable.rs

```diff
@@ -54,12 +76,41 @@ impl Executable {
 
     /// Stops the executable and returns the [ExitStatus].
     /// If the executable has never been started, returns [None].
-    pub fn kill(&mut self) -> Result<Option<ExitStatus>, io::Error> {
+    pub fn kill(&mut self) -> io::Result<Option<ExitStatus>> {
         match &mut self.state {
             ExecutableState::Init { .. } => Ok(None),
             ExecutableState::Started { process, .. } => {
                 process.kill(Some(SIGKILL))?;
                 let exit_status = process.wait()?;
+                // TODO
+                //self.log_thread
+                //    .unwrap()
+                //    .join()
+                //    .expect("logging thread panicked")?;
+                for line in self.read_stdout()? {
+                    event!(
+                        Level::INFO,
+                        level = "info",
+                        channel = self.stdout.name(),
+                        line
+                    );
+                    LogChannel::log_line(
+                        self.stdout.get_producer().clone(),
+                        line.to_string(),
+                    );
+                }
+                for line in self.read_stderr()? {
+                    event!(
+                        Level::INFO,
+                        level = "error",
```

Comment:
I think you meant to have `Level::INFO` and `level = "error"` but commenting just in case.

Reply:
i went back and forth.. it felt weird to have errors in the server logs caused by user script output, so i settled on info level logging with an inner "level" to indicate the inner logging level.  i could also just have "channel" or "stream".

Reply:
we now have inner level and channel and a span to hold them, which i think means "info" is ok.